app/helpers/application_helper/toolbar/ems_containers_center.rb

```diff
@@ -19,10 +19,16 @@ class ApplicationHelper::Toolbar::EmsContainersCenter < ApplicationHelper::Toolb
         button(
           :ems_container_new,
           'pficon pficon-add-circle-o fa-lg',
-          t = N_('Add a New #{ui_lookup(:table=>"ems_container")}'),
+          t = N_('Add Existing #{ui_lookup(:table=>"ems_container")}'),
           t,
           :url => "/new"),
         button(
+          :ems_container_deployment,
+          'pficon pficon-add-circle-o fa-lg',
+          t = N_('Create #{ui_lookup(:table=>"ems_container")}'),
+          t,
+          :data => {'function' => 'miqCallAngular', 'function-data' => '{ "name": "showListener", "args": [] }'}),
```

Comment:
Isn't `showListener` a very / too generic name?
Would it make sense to somehow limit the namespace of this to the container provider so that other providers/MiQ platform could have a function with the same name?